src/app/ipv/middleware.js

```diff
@@ -389,6 +392,18 @@ module.exports = {
       next(error);
     }
   },
+  handleConfirmDeleteDetailsAction: async (req, res, next) => {
+    try {
+      if (req.body?.journey === "reset") {
+        await handleJourneyResponse(req, res, "journey/reset");
```

Comment:
this can just be journey/next as there are only two options meaning we can reuse the handleJourneyAction. The subsequent change will need to be made in the journey map to use next. We've done that elsewhere for example on the `PYI_F2F_TECHNICAL`

Reply:
Ah yep good spot